### DISCUSSION TITLE
Lang tools docs

### DISCUSSION THREAD

**Discussion by davidmatter** [vuejs/language-tools#4169] [Ideas]
@johnsoncodehk I've watched the issues that've been created over the last couple of weeks and realised that it would probably make sense to create a (vitepress) documentation site for the Vue language tools as there are quite a few things one needs to know about when using the LSP/extension or when updating from 1.x to 2.x. I imagine that this could massively reduce the number of issues created and make your life a bit easier.

I'm offering to setup an initial version of such a site. What do you think?
Everyone, please comment on what kind of content/sections would help.

Some initial ideas:
- FAQ
- Some details about hybrid mode, how it works and what the tradeoffs are
- Details about insiders edition
- Roadmap
- nvim config guide
- Details about using the extension in VSCode 1.87/1.88 regarding the Node bug
- Details about the typechecking cli

**Comment by johnsoncodehk** [comment]
Thank you for your suggestion! I do intend to improve the documentation, but probably through the Readme/Github Wiki rather than a separate documentation site, I think this will ensure that it remains agile.
